### PR TITLE
docs(topics): add storage document

### LINKED ISSUE #14: doc topic: Configuring Persistent Volumes and Storage Clases
<a href="https://github.com/technosophos"></img></a> **Issue by [technosophos](https://github.com/technosophos)**
_2017-08-28 23:34:17_

----

This section would need to cover the different PV/Storage Classes that Acid can use.

Right now:

- Everything works out of the box with minikube
- On ACS, you need to create Azure Storage and configure an Azure Storage PV/Storage class

Everything else is untested.

### PR DESCRIPTION
**Pull request by technosophos**
This explains how Brigade uses storage.

Closes #14

### CODE REVIEW

File: docs/topics/storage.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# How Brigade uses Kubernetes Persistent Storage
+
+Brigade allows script authors to declare two kinds of storage:
+
+- per-job caches, which persist across builds
+- per-build shared storage, which exists as long as the build is running
+
+Usage of these is described within the [javascript.md](JavaScript docs) and the
```
Comment (bacongobbler) [inline docs/topics/storage.md]: need to add link to the javascript docs here

File: docs/topics/storage.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Brigade provisions storage using Kubernetes PVCs. Both caches and shared storage
+are PVC-backed.
+
+### Caches
+
+For a Cache, the Brigade worker will check to see if a Job asks for a cache. If it
+does, the worker will create a PVC (if it doesn't already exist) and then mount
+it to the cache.
+
+> A Job, in this case, gains its identity from its name, and the project that
```
Comment (bacongobbler) [inline docs/topics/storage.md]: Is there a particular reason this paragraph is quoted via right angle brackets (>)?

Comment (technosophos) [reply docs/topics/storage.md]: Isn't that how you do callouts in MD?

Comment (bacongobbler) [reply docs/topics/storage.md]: Yes it is. I just wanted to make sure that this was intentional. If so, carry on!

**Pull request merged**